gmso/external/convert_parmed.py

```diff
@@ -3,6 +3,7 @@
 
 import numpy as np
 import unyt as u
+from matplotlib.widgets import EllipseSelector
```

Comment:
What is this used for?

Reply:
I might have added that there by accident, deleted